infrastructure/vagrant/README.md

```diff
@@ -75,3 +75,19 @@ vagrant halt
 cd ~/triplea/infrastructure/vagrant
 vagrant destroy -f
 ```
+
+## Troubleshooting
+
+### VM Won't Start
+
+- Open VirtualBox, the VM should be listed there and you can open a console to 
```

Comment:
"open a console to the VM" sounds a little bit odd. Maybe "a console _of_ the VM" is better?